R/describe_posterior.R

```diff
@@ -218,11 +218,13 @@ describe_posterior.stanreg <- function(posteriors, centrality = "median", disper
     out <- out[match(col_order, out$Parameter), ]
   }
 
-  if (priors) {
-    col_order <- out$Parameter
-    priors_data <- describe_prior(posteriors, ...)
-    out <- merge(out, priors_data, all = TRUE)
-    out <- out[match(col_order, out$Parameter), ]
+  if (!is.null(priors)){
```

Comment:
You could use `isTRUE(priors)` here to capture all exceptions (null and true).

Reply:
Thanks, didn't know that 😉